bciers/apps/administration/app/components/userOperators/SelectOperatorConfirmForm.tsx

```diff
@@ -0,0 +1,160 @@
+"use client";
+
+import {
+  Box,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableRow,
+} from "@mui/material";
+import { useState } from "react";
+import WarningIcon from "@mui/icons-material/Warning";
+import RequestAccessButton from "../buttons/RequestAccessButton";
+import Link from "next/link";
+import { BC_GOV_LINKS_COLOR, DARK_GREY_BG_COLOR } from "@bciers/styles/colors";
+import Button from "@mui/material/Button";
+import { Operator } from "./types";
+
+interface ConfirmSelectedOperatorFormProps {
+  operator: Operator;
+  hasAdmin: boolean;
+}
+
+export default function SelectOperatorConfirmForm({
+  operator,
+  hasAdmin,
+}: Readonly<ConfirmSelectedOperatorFormProps>) {
+  const [hasConfirmedOperator, setHasConfirmedOperator] = useState(false);
+
+  const operatorHasAdminJSX: JSX.Element = (
+    <div data-testid="has-admin-message">
+      <p>
+        Looks like you do not have access to <b>{operator.legal_name}</b>.
+      </p>
+      <p>
+        An Operation Representative with Administrator access will need to
+        approve your access request.
+      </p>
+      <p>Please confirm below if you would like to submit an access request.</p>
+      <RequestAccessButton
+        operatorId={operator.id}
+        operatorName={operator.legal_name}
+      />
+    </div>
+  );
+
+  const operatorHasNoAdmin: JSX.Element = (
+    <div data-testid="has-no-admin-message">
+      <p>
+        Looks like operator <b>{operator.legal_name}</b> does not have
+        Administrator access set up.
+      </p>
+      <p>
+        Would you like to request Administrator access as an Operation
+        Representative?
+      </p>
+      <p>
+        Please note that you will be responsible for approving any additional
+        users requesting access to the operator.
+      </p>
+      <RequestAccessButton
+        operatorId={operator.id}
+        operatorName={operator.legal_name}
+        isAdminRequest={true}
+      />
+    </div>
+  );
+
+  return (
+    <section className="text-center my-auto text-2xl flex flex-col gap-3">
+      {hasConfirmedOperator ? (
+        <>
+          <span>
+            <WarningIcon sx={{ color: "#ff0e0e", fontSize: 50 }} />
+          </span>
+          <div>{hasAdmin ? operatorHasAdminJSX : operatorHasNoAdmin}</div>
+          <Link
+            href="#"
+            className="underline hover:no-underline"
+            style={{ color: BC_GOV_LINKS_COLOR }}
+            onClick={() => setHasConfirmedOperator(false)}
+          >
+            Go Back
+          </Link>
+        </>
+      ) : (
+        <>
+          <p>Kindly confirm if this is the operator that you represent.</p>
+          <Box display="flex" justifyContent="center">
+            <TableContainer
```

Comment:
For your consideration - I'm not going to ask you to redo this whole thing, however we did have a team agreement to avoid usage of MUI`Box` because why abstract basic html elements and tie it to a library in our beautiful open source project. So maybe consider avoiding usage of MUI for basic html elements in the future? 😅 

Though even if I'm not a fan I understand that using the table stuff was likely easier to get going.

Reply:
@marcellmueller Made a [debt tech ticket](https://github.com/bcgov/cas-registration/issues/2151) to update this component, since in this PR it was a simple copy/paste from Reg1,